CHANGELOG.md

```diff
@@ -4,11 +4,35 @@
 
 ### Fixes
 
+## 3.11.0 (Feature release)
+
+### New features
+
+#### Allow users to accept or reject cookies with a cookie banner
+
+You can now [use cookie banners](https://design-system.service.gov.uk/components/cookie-banner/) to allow users to accept or reject cookies which are not essential for making your service work.
+This was added in pull request #?: Add cookie banner.
```

Comment:
We need a link and PR number here.

Reply:
https://github.com/alphagov/govuk-frontend/pull/2131